src/pool/reserve_types.ts

```diff
@@ -442,78 +442,89 @@ export class ReserveEmissionData extends EmissionData {
 export class ReserveEmissions {
   constructor(
     public poolId: string,
-    public assetId: string,
-    public supplyEmissions: Emissions | undefined,
-    public borrowEmissions: Emissions | undefined
+    public supplyEmissions: Map<string, Emissions>,
+    public borrowEmissions: Map<string, Emissions>
```

Comment:
this seems over-optimized, as this object is quite confusing now.

IMO we should move forward with the V1 reserve loading optimization only